lib/web_ui/test/canvaskit/common.dart

```diff
@@ -21,7 +21,7 @@ const MethodCodec codec = StandardMethodCodec();
 /// Common test setup for all CanvasKit unit-tests.
 void setUpCanvasKitTest() {
   setUpAll(() async {
-    expect(useCanvasKit, true, reason: 'This test must run in CanvasKit mode.');
+    expect(renderer is CanvasKitRenderer, true, reason: 'This test must run in CanvasKit mode.');
```

Comment:
You could also use the `isA` matcher:
```suggestion
    expect(renderer, isA<CanvasKitRenderer>, reason: 'This test must run in CanvasKit mode.');
```